src/main/java/cheatsheet/CheatSheetList.java

```diff
@@ -0,0 +1,90 @@
+package cheatsheet;
+
+import java.util.ArrayList;
+
+public class CheatSheetList {
+    private ArrayList<CheatSheet> cheatSheets = new ArrayList<>();
+
+    public CheatSheetList() {
+    }
+
+    /**
+     * Gets the number of cheat sheet currently in the CheatSheetList.
+     *
+     * @return size of cheatSheets
+     */
+    public int getSize() {
+        return cheatSheets.size();
+    }
+
+    /**
+     * Gets all the cheat sheets in the cheatSheetList.
+     *
+     * @return cheatSheets The current list of cheat sheets
+     */
+    public ArrayList<CheatSheet> getCheatSheetList() {
+        return cheatSheets;
+    }
+
+    /**
+     * Prints the cheatSheetName attribute for each cheat sheet.
+     */
+    public void printCheatSheetNames() {
+        // todo: replace with UI.printCheatSheetNames
+        System.out.println("Current list of cheat sheet:");
+        for (CheatSheet cs : cheatSheets) {
+            System.out.println(cs.getCheatSheetName());
```

Comment:
The printing of the cheatsheet names should return a String. All the Sytem.out.println should be handled by only the UI class.